SUASSystem/SUASSystem/load_sd_card.py

```diff
@@ -54,21 +56,9 @@ def load_sd_card(send_image_filenames, location_log, interop_client_array):
                             least_time_difference = difference_in_times
                     drone_gps_location = location_log[closest_time_index]["current_location"]
 
-                    fly_zones = construct_fly_zone_polygon(interop_client_array)
-
-                    if (Point(drone_gps_location).within(fly_zones)) == False:
-                         continue
+                    if fly_zones.contains_point([drone_gps_location.get_lat(), drone_gps_location.get_lon()]) == 0:
+                        # not in range
```

Comment:
[Inline comments should be on the same line of the argument, condition, or function that's being executed/evaluated according to PEP8 standards.](https://www.python.org/dev/peps/pep-0008/#inline-comments)

`# not in range` should be moved to the end of Line 59. The same should be done for all other in-line comments.